common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -1094,6 +1094,7 @@ public final class LC {
     @Supported
     public static final KnownKey contact_ranking_enabled = KnownKey.newKey(true);
 
+    public static final KnownKey contact_allow_deref_group_member = KnownKey.newKey(true);
```

Comment:
The detail of contact group member is not shown if `contact_allow_deref_group_member` is false.
It is trade-off between contact group print functionality and performance. Performance issue may happen when lots of contact groups exists or contact group(s) has lots of members.
Which value is better as default?

Reply:
this LC is only used when we pass derefGroupMember = true in API, which we are doing in case of contact print only so I think it's ok to go ahead with this default value.